installer.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+if [[ "$#" -ne 1 ]]; then
+	echo "Usage: ./installer.sh <project_path>"
+    exit 1
+fi
+
+project_path=$1
+
+if ! command -v mvn &> /dev/null
+then
+    echo "First, you need Maven installed."
+    exit 1
+fi
+
+if ! command -v java &> /dev/null
+then
+    echo "First, you need Java installed."
+    exit 1
+fi
+
+echo "Compiling and installing BSP server..."
+mvn -q clean package
+jar_name="server-1.0-SNAPSHOT-spring-boot.jar"
+jar_path="$HOME/.bsp/maven/"
+mkdir -p "$jar_path"
+cp "server/target/$jar_name" "$jar_path"
```

Comment:
i believe that it would be better to provide a full path to the jar in the source project instead of storing it in the home dir

Reply:
I thought it would be safer to put it in the static home directory than project directory which might be changed by the user.

Reply:
thats true, i just dont like storing things in the home dir - the decision is yours